Dockerfile

```diff
@@ -1,185 +1,27 @@
-#                        |                                                          |
-#                        |                                                          |
-#              +---------+--------+                                                 |
-#              |                  |                                                 |
-#              |  builder-src     |<------------------------------+                 |
-#              |                  |                               |                 |
-#              +---+-----------+--+                               |                 |
-#                  ^           ^                                  |                 |
-#                  |           |                                  |                 |
-#                  |           |                                  |                 |
-#  +---------------+--+    +---+--------------+                                     |
-#  |                  |    |                  |                                     |
-#  |  builder-client  |    |  builder-server  |                                     |
-#  |                  |    |                  |                                     |
-#  +---------------+--+    +---+--------------+                                     |
-#                  ^           ^                                  ^                 |
-#                  |           |                                  |                 |
-#                  |           |                                  |                 |
-#              +---+-----------+--+                               |                 |
-#              |                  |-------------------------------+                 |
-#              |  builder         +<------------------------------------------+     |
-#              |                  |                                           |     |
-#              +--------+---------+                                           |     |
-#                       ^                                                     |     |
-#                       |                                                     |     |
-#                       |                                                     |     |
-#              +--------+---------+                                       +---+-----+--------+
-#              |                  |                                       |                  |
-#              |  electron-runner |                                       |  production      |
-#              |                  |                                       |                  |
-#              +------------------+                                       +------------------+
-
-# ---------------------------------------------------------
-# Base build dependencies
-# ---------------------------------------------------------
-FROM node:18.15.0-alpine3.17 as builder-base
+FROM node:18.15-alpine as builder-base
```

Comment:
I think one goal of the multistage builds are smaller image size (e.g. for production). I'm not sure if it's still a requirement, but it could be good information to compare the size before and after.

Reply:
And I agree it was kind of complicated. 😅

Reply:
> I think one goal of the multistage builds are smaller image size (e.g. for production). I'm not sure if it's still a requirement, but it could be good information to compare the size before and after.

Image size still important. If we can keep it [< 1GB](https://artsy.slack.com/archives/CA8SANW3W/p1726765549835789?thread_ts=1726761304.356259&cid=CA8SANW3W), we are good.

Reply:
Last we checked in slack the prod image was 400mb? We should double check this tho next later this week since I did adjust the docker file.